### ISSUE TITLE
iScroll 'scroll' event not working

### ISSUE THREAD

**Issue by grigorievd** [alvarotrigo/fullPage.js#3386]
### Description

I have a page working on fullpage.js with scrollOverflow: true, and the only thing is not working is 'scroll' event of iScroll, 'scrollEnd' and 'scrollStart' works correctly, I have "iscroll-probe.min.js" included before 'scrolloverflow.js' and probeType:3 as an option.

### Link to isolated reproduction with no external CSS / JS
https://codepen.io/anon/pen/RYaedQ

### Steps to reproduce it
https://codepen.io/anon/pen/RYaedQ

### Versions 
Latest Firefox, Chrome on Windows10

**Comment by alvarotrigo** [comment]
> I have "iscroll-probe.min.js" included before 'scrolloverflow.js' and probeType:3 as an option.

scrolloverflow is a forked version of iScroll that solves some issues that existed in iScroll when putting it together with fullPage.js.

You would have to replace (not insert before), scrolloverflow for iscroll-probe as detailed in [my answer here](https://stackoverflow.com/a/39369282/1081396) or as proposed [in this pull request](https://github.com/alvarotrigo/fullPage.js/pull/2875).

There's no proper solution for it provided by fullpage.js at the moment.

**Comment by grigorievd** [comment]
@alvarotrigo thanks for you answer, but when I'm replacing  scrolloverflow.js with  scrolloverflow for iscroll-probe I'm getting this error "fullPage: The option `scrollOverflow:true` requires the file `scrolloverflow.min.js`. Please include it before fullPage.js.".
Is it possible to workaround this ?

**Comment by alvarotrigo** [comment]
Yeah sorry about that, you would then need to add [the fullPage.js part](https://github.com/alvarotrigo/fullPage.js/blob/master/vendors/scrolloverflow.js#L2127) from line 2127 onwards.

**Comment by alvarotrigo** [comment]
I'm talking about fullpage.js version 3, of course :)
